bin/hostname/tests/hostname_test.sh

```diff
@@ -0,0 +1,57 @@
+#-
+# Copyright (c) 2023 Klara, Inc.
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+#
+# These tests need to run in a multibyte locale with non-localized
+# error messages.
+#
+export LC_CTYPE=C.UTF-8
+export LC_MESSAGES=C
+
+root_dir=$(atf_get_srcdir)
+
+test_jail_conf='test-hostname-jail {
+    host.hostname = "test-hostname";
+    path = "/";
+    persist;
+}'
+
+init()
+{
+    echo "$test_jail_conf" > "$root_dir/jail.conf"
```

Comment:
For this and the following ones, shell variables should be quoted by braces. i.e., `${test_jail_conf}` here.

Reply:
And since the test_jail_conf variable is only used once. Perhaps we can do something like this:

```
cat > "${root_dir}/jail.conf" <<EOF
test-hostname-jail {
    host.hostname = "test-hostname";
    path = "/";
    persist;
}
EOF
```

Reply:
Personally I don't like this way to code. I would break indent rule usually.